storage/src/qmdb/current/ordered.rs

```diff
@@ -171,14 +171,17 @@ impl<
                     return false;
                 }
 
-                (op_proof, Operation::Update(data))
+                (op_proof, Operation::Update(data.clone()))
```

Comment:
I suppose we should wait to find out how to clone `data` and `metadata` until we prove it is a bottleneck?